src/SprykerEco/Yves/Easycredit/Plugin/Provider/EasycreditControllerProvider.php

```diff
@@ -10,8 +10,14 @@
 use Silex\Application;
 use SprykerShop\Yves\ShopApplication\Plugin\Provider\AbstractYvesControllerProvider;
 
+/**
+ * @deprecated Use {@link \SprykerEco\Yves\Easycredit\Plugin\Router\EasycreditRouteProviderPlugin} instead.
```

Comment:
Please adjust changelogs by moving `Deprecated EasycreditControllerProvider.` into the `Deprecation` section. Usually, change logs contain either the `Improvements` or `Fixes` section, but it's not related to `Deprecation` - it should be a separate section.